tests/conftest.py

```diff
@@ -1674,12 +1674,40 @@ def tmpvenv_active_inherit(deactive_venv, tmpdir: py.path.local) -> Iterator[env
     loader.unload_modules_for_path(venv.site_packages_dir)
 
 
+@pytest.fixture(scope="session")
```

Comment:
Why is this a session scoped fixture? This way we will only cleanup the indexes at the end of the run of the entire test suite. By making it function scoped we cleanup after ever test. That way we are more efficient on disk-space and memory usage.

Reply:
But we do become faster because we don't have to rebuild the index every time?

Reply:
(ignore that, it is a non-caching factory)